src/drivers/nuopc/cime_driver/esm.F90

```diff
@@ -327,8 +321,8 @@ subroutine SetRunSequence(driver, rc)
     !   file=__FILE__)) &
     !   return  ! bail out
 
-!    call pretty_print_nuopc_freeformat(runSeqFF, 'run sequence', rc=rc)
-!    if (shr_nuopc_methods_ChkErr(rc,__LINE__,u_FILE_u)) return
+    ! call pretty_print_nuopc_freeformat(runSeqFF, 'run sequence', rc=rc)
```

Comment:
This was the original way. I have no idea why we want or do not want the pretty_print_nuopc. Frankly, I don't know what this produces. I am happy to take them out at this point since we are not using these lines.